ts/util/table-builder.ts

```diff
@@ -125,9 +125,10 @@ export class TableBuilder<ParentType, ChildType> {
     this.addPagingControls(topControlsContainer);
     this.container.appendChild(topControlsContainer);
 
+    const tableWrapper = document.createElement("div");
+    tableWrapper.className = "relative";
```

Comment:
Please add a comment to explain why this is necessary (separating `relative` and `overflow:hidden` to allow the tooltips to overflow without additional scrollbars)